bedrock/sitemaps/views.py

```diff
@@ -14,7 +14,7 @@ class SitemapView(TemplateView):
     content_type = "text/xml"
 
     def _get_locale(self):
-        if self.kwargs["is_none"]:
+        if "is_none" in self.kwargs:
```

Comment:
Is this because `is_none` can no longer be guaranteed to be available, but before it always was – just with a falsey value?

Reply:
Right, previously it would send `{"is_none": None}`, but after the `re_path` change the keys who's value is `None` were no longer included.